cli/src/commands/operation/diff.rs

```diff
@@ -124,42 +124,40 @@ pub fn cmd_op_diff(
         workspace_env.parse_template(&language, &text, CommitTemplateLanguage::wrap_commit)?
     };
 
+    let from_op_summary_template = {
+        let workspace_env = workspace_command.env();
+        let language = OperationTemplateLanguage::new(
+            from_op.op_store().root_operation_id(),
+            Some(from_op.id()),
```

Comment:
same here. `repo.op_id()`, and we don't need two separate template instances.

It might be better to add `workspace_command.operation_template_language()` as there are 3 callers.

Reply:
https://github.com/martinvonz/jj/pull/4471